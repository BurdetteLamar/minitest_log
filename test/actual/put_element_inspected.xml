<log>
  <no_inspected/>
  <one_inspected>
    :one
  </one_inspected>
  <more_inspected>
    :one23.140..2
  </more_inspected>
</log>
