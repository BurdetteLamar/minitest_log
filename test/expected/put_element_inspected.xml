<log>
  <summary verdicts='1' failures='0' errors='0'/>
  <no_inspected/>
  <one_inspected>
    :one
  </one_inspected>
  <more_inspected>
    :one23.140..2
  </more_inspected>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
